Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/unstructured-io/base-images:wolfi-base-e48da6b@sha256:8ad3479e5dc87a86e4794350cca6385c01c6d110902c5b292d1a62e231be711b as base
+FROM quay.io/unstructured-io/base-images:wolfi-base-e48da6b@sha256:f495978575bab94c2e20d09f286d73af55ae8ea83a255f5eb8794121b70f30fb as base
```

Comment:
What about using `FROM quay.io/unstructured-io/base-images:wolfi-base-latest as base`​ like in https://github.com/Unstructured-IO/core-product/blob/main/unstructured-api/Dockerfile#L5? So we don't have to update the base image every time whenever it's updated?

Reply:
Yeah that's a good call. Just updated to `latest` with no hash. And if `latest` ever breaks the build for whatever reason, we can alway pin to an older version.